lua/lvim/plugins.lua

```diff
@@ -158,8 +158,10 @@ local core_plugins = {
     config = function()
       require("lvim.core.which-key").setup()
     end,
-    event = "VeryLazy",
+    event = {"User FileOpened"},
+    keys = {"<leader>"},
```

Comment:
```suggestion
    event = "VeryLazy",
```
whichkey should be always loaded on startup, it works for more keys than just leader